tests/README.md

```diff
@@ -14,9 +14,9 @@ The credentials for `rktbot` are currently managed by CoreOS.
 Use the following build commands:
 
 ```
-./tests/install-deps.sh      # Setup
-./build                      # Setup
-./test                       # Thread #1
+./tests/install-deps.sh          # Setup
+./build                          # Setup
+DELETE_DATA_AND_CONFIG=1 ./test  # Thread #1
```

Comment:
Since Semaphore is not configured with that variable, this new test is not executed at the moment:
https://semaphoreci.com/coreos/rkt/branches/pull-request-760/builds/3

Instead of requiring a change in the Semaphore config, maybe you can just enable it for Semaphore like this:
```
diff --git a/test b/test
index f7213a2..7880bee 100755
--- a/test
+++ b/test
@@ -59,6 +59,8 @@ fi
 if [ "$CI" == true ] ; then
        if [ "${CIRCLECI}" == true -o "${SEMAPHORE}" == true ] ; then
                echo "Checking functional tests..."
+               # Run destructive tests too
+               export DELETE_DATA_AND_CONFIG=1
                (cd tests && ./test)
        else
                echo "Functional tests disabled."
```

Also, I wonder if the name `$RKT_ENABLE_DESTRUCTIVE_TESTS` would make it more clear.